src/components/calcite-tree-item/calcite-tree-item.scss

```diff
@@ -87,6 +87,7 @@
   text-decoration: none !important;
   max-width: 100%;
   word-wrap: break-word;
+  word-break: break-word;
```

Comment:
I think we may actually want `overflow-wrap: break-word;` here, from [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-wrap):

> Note: In contrast to word-break, overflow-wrap will only create a break if an entire word cannot be placed on its own line without overflowing.

They've actually marked `word-break: break-word` as deprecated.

Reply:
Hmmm... I wasn't able to get `overflow-wrap` to work on any combination of wrapping elements 🤔

Reply:
one second let me take a gander here

Reply:
weird I think it only works if you also give it `display: block` or `display: inline-block`